src/appengine/handlers/external_update.py

```diff
@@ -23,6 +24,9 @@
 from libs import handler
 from libs import helpers
 
+OLD_PROTOCOL = "1"  # Old message format: Data is a stacktrace.
```

Comment:
nit: use single quotes for all strings.

Reply:
Fixed in [9ef80a7](https://github.com/google/clusterfuzz/pull/2555/commits/9ef80a7a864ff6bd1731b0da3c80cd6901b3dacf).